src/Uno.UWP/UI/Composition/SkiaGeometrySource2D.skia.cs

```diff
@@ -2,19 +2,27 @@
 
 using SkiaSharp;
 using System;
-using System.Collections.Generic;
-using System.Text;
 using Windows.Graphics;
 
 namespace Windows.UI.Composition
 {
 	public class SkiaGeometrySource2D : IGeometrySource2D
 	{
-		public SkiaGeometrySource2D(SKPath? source = null)
+		public SkiaGeometrySource2D()
 		{
 			Geometry = new SKPath();
 		}
 
-		public SKPath? Geometry { get; }
+		public SkiaGeometrySource2D(SKPath source)
+		{
+			if (source == null)
+			{
+				throw new ArgumentNullException(nameof(source));
+			}
+
+			Geometry = source;
```

Comment:
```suggestion
			Geometry = source ?? throw new ArgumentNullException(nameof(source));
```

I would recommend you to use the more concise syntax when possible :)